fmn/fmn_fasshim.py

```diff
@@ -23,27 +25,45 @@
 default_url = 'https://admin.fedoraproject.org/accounts/'
 creds = config.app_conf['fas_credentials']
 
-fasclient = fedora.client.fas2.AccountSystem(
-    base_url=creds.get('base_url', default_url),
-    username=creds['username'],
-    password=creds['password'],
-)
+fasjson = config.app_conf['fasjson']
+if fasjson.get('active'):
+    client = Client(url=fasjson.get('url', default_url))
+else:
+    client = fedora.client.fas2.AccountSystem(
+        base_url=creds.get('base_url', default_url),
+        username=creds['username'],
+        password=creds['password'],
+    )
+
+
+def make_fasjson_cache(**config):
+    log.warning("Building the FASJSON cache into redis.")
+    if _cache.get('fas_cache_built'):
+        log.warning("FASJSON cache already built into redis.")
+        return
+    global client
+    try:
+        _add_to_cache(list(client.list_all_entities("users")))
```

Comment:
From a memory point of view it would be better to loop over `list_all_entities` than building the list in memory. Something like:
```
for users in client.list_all_entities("users"):
    _add_to_cache(users)
```

But that's not a blocker.